lms/templates/courseware/gradebook.html

```diff
@@ -118,7 +118,23 @@
         </tbody>
       </table>
     </div>
-
+    <span class="grade-book-footer">
+        %if page["previous_offset"] is not None:
+          <a href="${page_url}?offset=${page['previous_offset']}"
+             class="sequence-nav-button button-previous">
+            <span class="icon fa fa-chevron-left" aria-hidden="true"></span><span class="sr">${_('previous')}</span>
```

Comment:
I think it would be better to say "Previous Page" than just "previous" to give more context as to where the button takes you.